randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -1674,7 +1680,11 @@ Extra - in_dark_aether: True
           Dark World Damage ≥ 15 or Activate Safe Zone
           Any of the following:
               Dark World Damage ≥ 15 and Use Screw Attack (Space Jump)
-              Morph Ball and Space Jump Boots and Roll Jump (Hypermode) and Standable Terrain (Expert) and Dark World Damage ≥ 70
+              All of the following:
+                  Morph Ball and Space Jump Boots and Roll Jump (Hypermode) and Standable Terrain (Expert) and Dark World Damage ≥ 70
+                  All of the following:
+                      # https://www.youtube.com/watch?v=0k8JzNxnZLw
+                      Scan Visor and Combat/Scan Dash (Intermediate)
```

Comment:
This is nested under the wrong group.

Reply:
Hopefully correct now.

Reply:
It's still nested incorrectly.

Reply:
Should be good now

Reply:
And immediately after I pushed I realized it wrote the wrong room name, sick.